test/regress/regress0/bv/bv-int-collapse1.smt2

```diff
@@ -1,4 +1,4 @@
-; COMMAND-LINE: --bv-solver=lazy
+; COMMAND-LINE: --solve-bv-as-int=sum
```

Comment:
how about adding this line while keeping the previous one instead of removing the previous one? Same for other tests here.

Reply:
I'm planning to delete our prior support for lazy handling of bv2nat in the lazy solver, so this is intentional.